src/main/flight/gps_rescue.c

```diff
@@ -112,6 +112,8 @@ typedef struct {
     float alitutudeStepCm;
     float maxPitchStep;
     float absErrorAngle;
+    float groundspeedPitchAttenuator;
+    float imuYawGain;
```

Comment:
```suggestion
    float imuCoGYawGain;
```
(sorry, missed that before. Only CoG is affected, other sources of Yaw are unchanged (= compass)

Reply:
Nice.  Done.